app/src/components/Register.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import axios from 'axios'
+
+class Register extends React.Component {
+  state = {
+    username: '',
+    password: '',
+    department: ''
+  }
+  inputHandler = e => {
+    let value = e.target.value
+    this.setState({ [e.target.name]: value })
+  }
+  submit = e => {
+    e.preventDefault()
+    axios
+      .post(`http://localhost:9090/api/users/register`, {
+        username: this.state.username,
+        password: this.state.password,
+        department: this.state.department
+      })
+      .then(res => {
+        console.log(res.data)
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }
+  render() {
+    return (
+      <div>
+        <form onSubmit={this.submit}>
+          <div>
+            Username:
+            <input
+              type="text"
+              name="username"
+              onChange={this.inputHandler}
+              value={this.state.username}
+            />
+          </div>
+          <div>
+            Password:
+            <input
+              type="password"
+              name="password"
+              onChange={this.inputHandler}
+              value={this.state.password}
+            />
+          </div>
+          <div>
+            Department:
+            <input
+              type="text"
+              name="department"
+              onChange={this.inputHandler}
+              value={this.state.department}
+            />
```

Comment:
These input divs look super similar. Look to [this review comment](https://github.com/edreeseg/webauth-i-challenge/pull/1#discussion_r270271307) for some inspiration if you're interested in reducing the repetition.